### DISCUSSION TITLE
General Funtionality Of The Application

### DISCUSSION THREAD

**Discussion by roberto-morado** [careerJumpstart/TestimonialFree#20] [Ideas]
Post ideas about the functionality of the application

**Comment by ClaraLopezUribe** [comment]
Logins/authentications & authorizations for Reviewer

Should authentication process be required to leave reviews?

Down sides: 

-  Adds a point of friction that may discourage participation

- Increases data to manage and keep secure (consider 3rd party authentication or secured link given by the Receiver???)

Benefits:  

- Minimizes potential security concerns (bots/spam, no accountability for slander/inappropriate content, etc)

- Could this allow/facilitate the ability to display all of the testimonials written by a single Reviewer (similar to Google).
